HW0/P4/P4.py

```diff
@@ -55,8 +68,28 @@
     # Initial conditions for s0
     # Compute the rest of sk using Eq (1)
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '-k', label='Blind trajectory')
+    A = np.matrix([[1, 0, 0, dt, 0, 0],
+                   [0, 1, 0, 0, dt, 0],
+                   [0, 0, 1, 0, 0, dt],
+                   [0, 0, 0, 1 - c*dt, 0, 0],
+                   [0, 0, 0, 0, 1 - c*dt, 0],
+                   [0, 0, 0, 0, 0, 1 - c*dt]])
+
+    a = np.matrix([[0, 0, 0, 0, 0, g*dt]]).T
+
+    s0 = np.matrix('0 0 2 15 3.5 4.0').T
+
+    result = np.zeros((6, len(strue)))
```

Comment:
Consider using more descriptive variable names.  Result is a little vague without knowing its context